scannerazuresc.properties

```diff
@@ -11,7 +11,7 @@ publicVersions=3.0.2
 
 3.0.2.description=Fix .NET Framework scanner embedding logic.
```

Comment:
would it be the same description than above? Fix windows path parsing coming from predefined variables correctly in extraProperties cc @denis-troller